src/e2e/resources/data/AdminAccountsPageE2ETest.json

```diff
@@ -1,5 +1,12 @@
 {
   "accounts": {
+    "AAccounts.instr": {
```

Comment:
The reason the test passes in the dev server but not in staging is because very likely in staging the admin is also an instructor. It is because of the wrongly configured logic here:
https://github.com/TEAMMATES/teammates/blob/dda027eb5200cdf4fdf99d12374c27aa54f47a67/src/main/java/teammates/ui/webapi/DeleteInstructorAction.java#L57-L58